src/modules/services/main.py

```diff
@@ -35,37 +38,64 @@ def run():
 
     # enable services
     for svc in services:
-        ec = libcalamares.utils.target_env_call(['systemctl', 'enable', '{}.service'.format(svc['name'])])
+        ec = libcalamares.utils.target_env_call(
+            ['systemctl', 'enable', '{}.service'.format(svc['name'])]
+            )
 
         if ec != 0:
             if svc['mandatory']:
-                return "Cannot enable systemd service {}".format(svc['name']), \
-                       "systemctl enable call in chroot returned error code {}".format(ec)
+                return ("Cannot enable systemd service {}".format(svc['name']),
+                        "systemctl enable call in chroot returned error code",
+                        "{}".format(ec)
```

Comment:
This changes it to a 3-tuple -- drop the extraneous comment, and re-add the space.